pypolychord/polychord.py

```diff
@@ -112,8 +112,8 @@ def run_polychord(loglikelihood, nDims, nDerived, settings,
 
         Parameters
         ----------
-        distance2_matrix: numpy.array
-            squared distances between points. Shape (nPoints, nPoints)
+        position_matrix: numpy.array
```

Comment:
Could we call this `points` rather than `position_matrix` (and change throughout)